addon/common/constants.js

```diff
@@ -1,6 +1,6 @@
-/* global require */
-const APP_VERSION = require('../../package.json').version;
+import { getOwnConfig } from '@embroider/macros';
+const APP_VERSION = getOwnConfig().version;
```

Comment:
Ed F's fix from the Embroider support issue seems to have fixed this when using babel@7 deps
https://github.com/imgix/ember-cli-imgix/issues/210#issuecomment-979689198